### PR TITLE
Add Profiles sample

### PR DESCRIPTION
**Pull request by vladislav-tsvek**

### CODE REVIEW

File: vbnet/Profiles.WinForms/BrowserForm.vb [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+Imports DotNetBrowser.WinForms
+
+Namespace Profiles.WinForms
+	Partial Public Class BrowserForm
+		Inherits Form
+
+		Private ReadOnly browserView As BrowserView
+		Private _browser As IBrowser
+
+		#Region "Properties"
```
Comment (evhenii-komar) [inline vbnet/Profiles.WinForms/BrowserForm.vb:35]: The region should be aligned to the left side. This file required code reformating by Resharper.

File: vbnet/Profiles.WinForms/Form1.vb [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+' DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+' THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+' (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+' OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+#End Region
+
+Imports System
+Imports System.Linq
+Imports System.Windows.Forms
```
Comment (evhenii-komar) [inline vbnet/Profiles.WinForms/Form1.vb:25]: Unused Imports. This file required Resharper reformating.

File: vbnet/Profiles.WinForms/Program.vb [hunk 3]
```diff
@@ -0,0 +1,42 @@
+﻿#Region "Copyright"
```
Comment (evhenii-komar) [inline vbnet/Profiles.WinForms/Program.vb:1]: Resharper reformating is required for this file.

**Review by evhenii-komar — CHANGES_REQUESTED** [review]

**Comment by vladislav-tsvek** [comment]
@evhenii-komar I have refactored the code. Please take a look.

**Review by evhenii-komar — APPROVED** [review]

**Pull request merged**